docusaurus/docs/dev-docs/intro.md

```diff
@@ -7,18 +7,27 @@ sidebar_position: 1
 
 # Welcome to the Strapi Developer Docs!
 
-This documentation contains all technical documentation related to the setup, deployment, update, and customization of your Strapi application.
+<SubtleCallout title="Developer Docs, User Guide, and Strapi Cloud documentation" emoji="📍">
 
-:::strapi Can't wait to start using Strapi?
-You can directly head to the [Quick Start](./quick-start.md)! <br/> If demos are more your thing, we have a [video demo](https://youtu.be/zd0_S_FPzKg), or you can request a [live demo](https://strapi.io/demo)!
-:::
+The documentation for Strapi contains 3 main sections, accessible from the top navigation bar:
+
+- 🧑‍🏫 The **[User Guide](/user-docs/intro)** is all about using Strapi's admin panel.
+- ☁️ The **[Strapi Cloud](/cloud/intro)** documentation is about deploying your Strapi application to Strapi Cloud and managing your Strapi Cloud projects and settings.
+- 🧑‍💻 The **Developer Docs** that you're currently reading contain all the technical information related to the setup, advanced usage, customization, and update of your Strapi v5 application.
+
+</SubtleCallout>
+
+👉 If you're totally new to **Strapi** <Annotation>**💡 Did you know?**<br />The original purpose of the project was to help Boot**strap** your **API**: that's where the Strapi name comes from and how Strapi was created.<br /><br />Now, Strapi is an **open-source headless CMS** that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel.<br /><br />Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible — and which every part is customizable to match any use case. Strapi also has a built-in user system to manage in detail what the administrators and end users have access to.<br /></Annotation> or if you're a developer who'd like to take the most out of Strapi, the Developer Docs are the place to start! We suggest you to:
+
+1. Head over to the [Quick Start](/dev-docs/quick-start) guide.
+2. Continue your journey through the various Developer Docs sections. The ["What you will learn"](#what-you-will-learn) section gives you an overview of the available topics.
 
-The original purpose of the project was to help Boot**strap** your **API**: that's how Strapi was created. Now, Strapi is an open-source headless CMS that gives developers the freedom to choose their favorite tools and frameworks and allows editors to manage and distribute their content using their application's admin panel. Based on a plugin system, Strapi is a flexible CMS whose admin panel and API are extensible - and which every part is customizable to match any use case. Strapi also has a built-in user system to manage in detail what the administrators and end users have access to.
 
-## How to use the Strapi Developer Docs
+
+## What you will learn
```

Comment:
Completely open question: should we name that title "What you will learn" or something like "What you can learn about"/"What you will find here"? Basically, should we choose a title that emphasise that the user's journey can be based on his choices and specific needs? I hope that makes sense.

Reply:
That's a good suggestion, thanks! I like the "What you will find here" even more because I feel like it emphasizes the journey ("here" vs. what is not here, if that makes sense 😄). I've updated it.